app/models/activity.rb

```diff
@@ -157,4 +157,12 @@ def transactions_total
   def planned_disbursements_total
     planned_disbursements.sum(:value)
   end
+
+  def actual_total_for_report(report:)
```

Comment:
I think I would like to name this 'actual_total_for_report_financial_quarter' - I feel like we should express what the date range is doing in it as best we can?